drivers/adc-dac/ad5592r/ad5592r-base.h

```diff
@@ -112,6 +112,7 @@ struct ad5592r_dev {
 	uint8_t gpio_out;
```

Comment:
Can you please add a short description in the commit about what is LDAC mode and why we need it?